src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -0,0 +1,190 @@
+                    isError
+                    message={error}
+                />
+            )}
+            <Button
+                success
+                large
+                text={translate('common.confirm')}
+                onPress={() => {
+                    if (userReportedIntegration === undefined) {
+                        setError(translate('onboarding.purpose.errorSelection'));
+                        return;
+                    }
+
+                    if (!onboardingPurposeSelected) {
+                        return;
+                    }
+
+                    Report.completeOnboarding(
+                        onboardingPurposeSelected,
+                        CONST.ONBOARDING_MESSAGES[onboardingPurposeSelected],
+                        undefined,
+                        undefined,
+                        onboardingAdminsChatReportID ?? undefined,
+                        onboardingPolicyID,
+                        undefined,
+                        onboardingCompanySize,
+                        userReportedIntegration,
+                    );
+
+                    Welcome.setOnboardingAdminsChatReportID();
+                    Welcome.setOnboardingPolicyID();
+
+                    Navigation.dismissModal();
+
+                    // Only navigate to concierge chat when central pane is visible
+                    // Otherwise stay on the chats screen.
+                    if (!shouldUseNarrowLayout && !route.params?.backTo) {
+                        Report.navigateToConciergeChat();
+                    }
+                }}
+                pressOnEnter
+            />
+        </>
+    );
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            testID="BaseOnboardingAccounting"
+            style={[styles.defaultModalContainer, shouldUseNativeStyles && styles.pt8]}
+        >
+            <HeaderWithBackButton
+                shouldShowBackButton
+                progressBarPercentage={75}
+                onBackButtonPress={OnboardingFlow.goBack}
+            />
+            <View style={[onboardingIsMediumOrLargerScreenWidth && styles.mt5, onboardingIsMediumOrLargerScreenWidth ? styles.mh8 : styles.mh5]}>
+                <Text style={[styles.textHeadlineH1, styles.mb5]}>{translate('onboarding.accounting.title')}</Text>
+            </View>
+            <SelectionList
+                sections={[{data: accountingOptions}]}
+                onSelectRow={(item) => {
+                    setUserReportedIntegration(item.keyForList);
+                    setError('');
+                }}
+                shouldUpdateFocusedIndex
+                ListItem={RadioListItem}
+                footerContent={footerContent}
+                shouldShowTooltips={false}
+                listItemWrapperStyle={onboardingIsMediumOrLargerScreenWidth ? [styles.pl8, styles.pr8] : []}
+            />
+            {shouldUseNarrowLayout && <OfflineIndicator />}
```

Comment:
Why is this needed? It's causing double offline indicator on small screens

<img width="468" alt="Screenshot 2024-10-08 at 10 48 20 AM" src="https://github.com/user-attachments/assets/b0c7cd5b-d50e-46ad-b31a-82d98bca6055">